config/Dockerfiles/runtest/Dockerfile_disabled

```diff
@@ -0,0 +1,46 @@
+FROM fedora:25
```

Comment:
what is this file for? it is named disabled...

Reply:
It is the Dockerfile that I believe we will use once we switch to container() calls. I had to make a second Dockerfile that is live and contains all the libvirt provisioning stuff, but this container shouldn't need that functionality so I didn't want to just trash my previous work

Reply:
smart chap